po/fr.po

```diff
@@ -238,6 +238,381 @@ msgstr "Pacman est en cours d'utilisation, veuillez patienter..."
+
+#: src/help.rs:30
+msgid "New options:"
+msgstr "Nouvelles options :"
+
+#: src/help.rs:31
+msgid "       --repo              Assume targets are from the repositories"
+msgstr "       --repo              Suppose que les cibles sont dans les depôts"
+
+#: src/help.rs:32
+msgid "    -a --aur               Assume targets are from the AUR"
+msgstr "    -a --aur               Suppose que les cibles sont dans l'AUR"
+
+#: src/help.rs:33
+msgid "    --aururl    <url>      Set an alternative AUR URL"
+msgstr "    --aururl    <url>      Défini une URL alternative pour l'AUR"
+
+#: src/help.rs:34
+msgid "    --clonedir  <dir>      Directory used to download and run PKGBUILDs"
+msgstr ""
+"    --clonedir  <dir>      Dossier utilisé pour télécharger et exécuter "
+"PKGBUILDs"
+
+#: src/help.rs:36
+msgid "    --makepkg   <file>     makepkg command to use"
+msgstr "    --makepkg   <file>     Commande makepkg à utiliser"
+
+#: src/help.rs:37
+msgid "    --mflags    <flags>    Pass arguments to makepkg"
+msgstr "    --mflags    <flags>    Passe des arguments à makepkg"
+
+#: src/help.rs:38
+msgid "    --pacman    <file>     pacman command to use"
+msgstr "    --pacman    <file>     Commande pacman à utiliser"
+
+#: src/help.rs:39
+msgid "    --git       <file>     git command to use"
+msgstr "    --git       <fichier>    Commande git à utiliser"
+
+#: src/help.rs:40
+msgid "    --gitflags  <flags>    Pass arguments to git"
+msgstr "    --gitflags  <flags>    Passe des arguments à git"
+
+#: src/help.rs:41
+msgid "    --sudo      <file>     sudo command to use"
+msgstr "    --sudo      <file>     Commande sudo à utiliser"
+
+#: src/help.rs:42
+msgid "    --sudoflags <flags>    Pass arguments to sudo"
+msgstr "    --sudoflags <flags>    Passe des arguments à sudo"
+
+#: src/help.rs:43
+msgid "    --asp       <file>     asp command to use"
+msgstr "    --asp       <file>     Commande asp à utiliser"
+
+#: src/help.rs:44
+msgid "    --bat       <file>     bat command to use"
+msgstr "    --bat       <file>     Commande bat à utiliser"
+
+#: src/help.rs:45
+msgid "    --batflags  <flags>    Pass arguments to bat"
+msgstr "    --batflags  <flags>    Passe des arguments à bat"
+
+#: src/help.rs:46
+msgid "    --gpg       <file>     gpg command to use"
+msgstr "    --gpg       <file>     Commande gpg à utiliser"
+
+#: src/help.rs:47
+msgid "    --gpgflags  <flags>    Pass arguments to gpg"
+msgstr "    --gpgflags  <flags>    Passe des arguments à gpg"
+
+#: src/help.rs:48
+msgid "    --fm        <file>     File manager to use for PKGBUILD review"
+msgstr ""
+"    --fm        <file>     Gestionnaire de fichier à utiliser pour la "
+"relecture des PKGBUILDs"
+
+#: src/help.rs:49
+msgid "    --fmflags   <flags>    Pass arguments to file manager"
+msgstr ""
+"    --fmflags   <flags>    Passe des arguments au gestionnaire de fichier"
+
+#: src/help.rs:51
+#, fuzzy
+msgid "    --completioninterval   <n> Time in days to refresh completion cache"
+msgstr ""
```

Comment:
Broken.

Reply:
This seems to be created by poedit. I removed it manually